src/shell/mpir/rangelist.c

```diff
@@ -25,16 +25,20 @@
  *   which is delta encoded from the previous entry (or 0 if this is the
  *   first entry in the rangelist).
  *
- * - If an entry is an array, it will have two elements. The fist element
```

Comment:
Same comment as before re: the commit message
- "commoon"
- extra space in the last bracketed element of the first example

Reply:
uhoh, I thought I had rebased after fixing those issues.. I wonder what else I missed :facepalm: ugh